### ISSUE TITLE
Currently installed pip packages in production (28.2.2024)

### ISSUE THREAD

**Issue by kulttuuri** [kulttuuri/jobeinabox#1]
As a reference, this is the list of currently installed pip packages in production, after SSHing into the jobe container and running pip3 list:

root@872c14bba3e5:/# pip3 list
Package                      Version
---------------------------- -------------------
absl-py                      1.0.0
astroid                      2.9.3
astunparse                   1.6.3
attrs                        21.4.0
autobahn                     17.10.1
Automat                      0.8.0
bcrypt                       3.1.7
beautifulsoup4               4.8.2
blis                         0.7.6
cachetools                   5.0.0
catalogue                    2.0.6
catkin-pkg                   0.4.24
catkin-pkg-modules           0.4.24
cbor                         1.0.0
certifi                      2021.10.8
charset-normalizer           2.0.12
click                        7.1.2
constantly                   15.1.0
cryptography                 2.8
cycler                       0.10.0
cymem                        2.0.6
Cython                       0.29.27
defusedxml                   0.6.0
distro                       1.4.0
docutils                     0.16
empy                         3.3.2
en-core-web-sm               3.0.0
flatbuffers                  2.0
gast                         0.5.3
gitdb                        4.0.10
GitPython                    3.1.29
google-auth                  2.6.0
google-auth-oauthlib         0.4.6
google-pasta                 0.2.0
graphviz                     0.20.1
greenlet                     1.1.2
grpcio                       1.43.0
h5py                         3.6.0
hyperlink                    19.0.0
idna                         3.3
imageio                      2.16.0
importlib-metadata           4.11.1
incremental                  16.10.1
iniconfig                    1.1.1
isort                        5.10.1
Jinja2                       3.0.3
joblib                       1.1.0
jsonlib-python3              1.6.1
keras                        2.8.0
Keras-Preprocessing          1.1.2
kiwisolver                   1.0.1
langcodes                    3.3.0
lazy-object-proxy            1.7.1
libclang                     13.0.0
lz4                          3.0.2+dfsg
Markdown                     3.3.6
MarkupSafe                   2.0.1
matplotlib                   3.1.2
mccabe                       0.6.1
mock                         4.0.3
mpi4py                       3.0.3
murmurhash                   1.0.6
netifaces                    0.10.4
networkx                     2.6.3
nltk                         3.7
nnpack                       0.1.0
nose                         1.3.7
numpy                        1.22.2
oauthlib                     3.2.0
opencv-python                4.5.5.62
opt-einsum                   3.3.0
packaging                    21.3
pandas                       1.4.1
paramiko                     2.6.0
pathy                        0.6.1
Pillow                       9.0.1
pip                          22.0.3
platformdirs                 2.5.0
pluggy                       1.0.0
preshed                      3.0.6
protobuf                     3.19.4
psutil                       5.5.1
py                           1.11.0
py-ubjson                    0.14.0
py4j                         0.10.9.5
pyarrow                      10.0.1
pyasn1                       0.4.8
pyasn1-modules               0.2.8
pycairo                      1.16.2
pycryptodomex                3.6.1
pydantic                     1.8.2
pydot                        1.4.1
PyHamcrest                   1.9.0
pylint                       2.12.2
PyNaCl                       1.3.0
PyOpenGL                     3.1.0
pyOpenSSL                    19.0.0
pyparsing                    2.4.6
PyQRCode                     1.2.1
PyQt5                        5.14.1
pyspark                      3.3.1
pytest                       7.0.1
python-dateutil              2.8.2
python-gnupg                 0.4.5
python-snappy                0.5.3
PyTrie                       0.2
pytz                         2021.3
PyWavelets                   1.2.0
PyYAML                       5.3.1
regex                        2022.1.18
requests                     2.27.1
requests-oauthlib            1.3.1
roman                        2.0.0
rosdep                       0.21.0
rosdep-modules               0.21.0
rosdistro                    0.8.3
rosdistro-modules            0.8.3
rosinstall                   0.7.8
rosinstall-generator         0.1.22
rospkg                       1.3.0
rospkg-modules               1.3.0
rsa                          4.8
runcython                    0.2.5
scikit-image                 0.19.1
scikit-learn                 1.0.2
scipy                        1.8.0
service-identity             18.1.0
setuptools                   45.2.0
sip                          4.19.21
six                          1.14.0
smart-open                   5.2.1
smmap                        5.0.0
soupsieve                    1.9.5
spacy                        3.0.7
spacy-legacy                 3.0.8
spacy-loggers                1.0.1
SQLAlchemy                   1.4.31
srsly                        2.4.2
tensorboard                  2.8.0
tensorboard-data-server      0.6.1
tensorboard-plugin-wit       1.8.1
tensorflow                   2.8.0
tensorflow-cpu               2.8.0
tensorflow-io-gcs-filesystem 0.24.0
termcolor                    1.1.0
tf-estimator-nightly         2.8.0.dev2021122109
thinc                        8.0.13
threadpoolctl                3.1.0
tifffile                     2022.2.9
toml                         0.10.2
tomli                        2.0.1
torch                        1.13.1+cpu
torchaudio                   0.13.1+cpu
torchview                    0.2.6
torchvision                  0.14.1+cpu
tqdm                         4.62.3
transferflow                 0.1.8
Twisted                      18.9.0
txaio                        2.10.0
typer                        0.3.2
typing_extensions            4.1.1
u-msgpack-python             2.1
urllib3                      1.26.8
vcstools                     0.1.42
wasabi                       0.9.0
Werkzeug                     2.0.3
wheel                        0.34.2
wrapt                        1.13.3
wsaccel                      0.6.2
wstool                       0.1.18
zipp                         3.7.0
zope.interface               4.7.1
WARNING: You are using pip version 22.0.3; however, version 24.0 is available.
You should consider upgrading via the '/usr/bin/python3 -m pip install --upgrade pip' command.
